### DISCUSSION TITLE
Issue forms: required should work for private repositories

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#63470] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

Currently, required is ignored for private repositories, regarding the documentation:
> Note: The required field key is only supported in public repositories. In private and internal repositories, all fields are optional.

I don't get the reason behind it. We need specific information for a feature request or a bug report and we can't rely that every member of our organization fills out every field. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by 0xBasically** [comment]
This would be nice to have

**Comment by awildeep** [comment]
I agrree with the OP.  A series of "No response" values is not an ideal situation when values are needed for triage/assessment/assignment/etc.

**Comment by Timmmy-nlb** [comment, marked as answer]
Good News: "required" is now working in Internal Repos!
https://github.com/orgs/community/discussions/148713#discussioncomment-12226996
